charts/sonatype-nexus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: sonatype-nexus
-version: 4.4.3
+version: 4.4.4
```

Comment:
Please bump to 5.0.0 as this may break existing deployments since it is changing labels on the storage related items

Reply:
Done